enrichments/trace/internal/elastic/span_test.go

```diff
@@ -364,6 +365,63 @@ func TestElasticTransactionEnrich(t *testing.T) {
 				return &spanLinks
 			}(),
 		},
+		{
+			name: "user_agent_parse_name_version",
```

Comment:
nit: I'd postfix these with something like `_span` and `_transaction`. So e.g. `user_agent_parse_name_version_transaction` here and `user_agent_parse_name_version_span` below, instead of having them with the same `name`.

Reply:
They would be reported as `TestElastic{Transaction, Span}Enrich/user_agent_parse_name_version`, is that not enough or is it for when looking at the tests individually without the context of the main test?

Reply:
Cool, I am thinking to simplify these tests with a method for default enrichments... currently the tests are getting a bit hard to read an necessarily verbose. Will do it later as a followup.